lib/searchable_select.ex

```diff
@@ -208,6 +213,7 @@ defmodule SearchableSelect do
       end
 
     selected = selected ++ [{key, val}]
+    if socket.assigns.send_change_events, do: send(self(), {:change, selected})
```

Comment:
1. A liveview with several searchable selects won't know for which one this is (rather also pass the `parent_key` as usual)

2. The `update_parent_view` called further down will, in most cases, still send the `:select` event. Sooo why not rather send a normal `{:select, parent_key, selected}` event there?
<img width="802" alt="image" src="https://github.com/user-attachments/assets/fe624cfe-be8d-4035-bf10-0d695fb9c1e7">
That eliminates the potential for receiving both the `:change` and the existing `:select` event in certain configurations.

Reply:
Thanks, this makes sense. Updated.